config/charts/inferencepool/templates/epp-deployment.yaml

```diff
@@ -181,13 +198,94 @@ spec:
         volumeMounts:
         - name: plugins-config-volume
           mountPath: "/config"
+      {{- if .Values.inferenceExtension.latencyPredictor.enabled }}
+      # Training Server Sidecar Container
+      - name: training-server
```

Comment:
can we define this sidecar somewhere else and we embed it here, inlining this whole thing directly here makes it hard to read